apis/net/v1alpha1/tunnel_endpoint_types.go

```diff
@@ -43,11 +45,14 @@ type TunnelEndpointStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file\
 	Phase                 string     `json:"phase,omitempty"` //two phases: New, Processed
-	LocalRemappedPodCIDR  string     `json:"localRemappedPodCIDR,omitempty"`
+	LocalNATPodCIDR       string     `json:"localNATPodCIDR,omitempty"`
 	LocalPodCIDR          string     `json:"localPodCIDR,omitempty"`
-	RemoteRemappedPodCIDR string     `json:"remoteRemappedPodCIDR,omitempty"`
-	OutgoingNAT           bool       `json:"outgoingNAT,omitempty"` // if true, the local podCIDR has been remapped by the remote cluster
-	IncomingNAT           bool       `json:"incomingNAT,omitempty"` // if true, the remote podCIDR has been remapped by the local cluster
+	RemoteNATPodCIDR      string     `json:"remoteNATPodCIDR,omitempty"`
+	LocalExternalCIDR     string     `json:"localExternalCIDR,omitempty"`     // External CIDR for remote cluster.
```

Comment:
Can you please add comments to be included in the CRD definition as a description field?

Reply:
Sure!